superset/assets/javascripts/explorev2/components/Filter.jsx

```diff
@@ -65,13 +84,23 @@ export default class Filter extends React.Component {
             onChange={this.changeOp.bind(this, this.props.filter)}
           />
           <div className="col-lg-6">
-            <input
+            {this.props.filter_select ?
```

Comment:
this might be easier to read if we pull it out into it's own method, like `renderFilterInput`